flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/consumer/InputBuffersMetricsTest.java

```diff
@@ -43,19 +45,18 @@
 import java.util.List;
 
 import static org.junit.Assert.assertEquals;
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
 
 /**
  * Test for input buffer usage related metrics.
  */
 public class InputBuffersMetricsTest {
 
+	private CloseableRegistry closeableRegistry = new CloseableRegistry();
```

Comment:
one new issue: you can not close the same `CloseableRegistry` more then once, so you would have to set it up/create new instance in `@org.junit.Before` method. 

nit: you could also close it in the `@After` method.

Reply:
I think the `closeableRegistry` do not have to share between test cases, so i move this to the method scope.